machines/bleShare/scan/scanMachine.ts

```diff
@@ -441,7 +442,11 @@ export const scanMachine =
                 },
               },
             },
-            navigatingToHistory: {},
+            navigatingToHistory: {
+              invoke: {
+                src: 'disconnect',
```

Comment:
Do we require wallet disconnect on QRLogin as we would not have started the connection with wallet?